src/codegen/cppcg.cpp

```diff
@@ -1585,8 +1587,20 @@ void CppCodeGenerator::GenConstructor(PObjectBase class_obj, const EventVector &
 {
 	m_source->WriteLn();
 	m_source->WriteLn( GetCode( class_obj, wxT( "cons_def" ) ) );
-	m_source->WriteLn( wxT( "{" ) );
-	m_source->Indent();
+    if ( class_obj->GetPropertyAsInteger(wxT("create_children")) != 0 ) {
+        m_source->WriteLn( wxT( "{" ) );
+        m_source->Indent();
+        m_source->WriteLn( wxT( "Create();" ) );
+        m_source->Unindent();
+        m_source->WriteLn( wxT( "}" ) );
+    } else {
+        m_source->WriteLn( wxT( "{" ) );
+        m_source->WriteLn( wxT( "}" ) );
+    }
```

Comment:
This looks like a pretty hacky attempt to get the second method defined, i didn't take a detailed look at the present code but i think it would be a cleaner way to create the method definition in a dedicated method